pages/index.vue

```diff
@@ -1,26 +1,278 @@
 <template>
   <v-container>
-    <h1>h1: home rendered</h1>
-    <h2>h2: let's test</h2>
-    <h3>h3: header typography</h3>
-    <p class="title">title: and typography</p>
-    <p class="title-caps">title caps</p>
-    <p class="subtitle">subtitle: in general</p>
-    <p>default</p>
-    <p class="caption">caption</p>
-    <Button>A default button</Button>
-    <Button secondary>A secondary button</Button>
-    <Button disabled large>A disabled button</Button>
-    <Button primary hero nuxt to="/support">A hero button w/ link</Button>
+    <v-row id="home">
+      <v-col>
+
+
+        <v-row id="hero">
+          
+
+          <!-- why isn't this cols="8" seeming to have an effect? -->
+          <v-col cols="8" class="px-10">
+
+            <div class="hero-copy">
+
+              <h1>Together, we have the power to stop the spread of COVID-19</h1>
+
+              <p class="subtitle mt-10">Protect the health for your family and community while preserving your privacy through mobile alerts.</p>
+
+              <div class="mt-10 d-flex flex-wrap">
+                <Button primary hero nuxt to="/about" class="mr-10 mb-4" >Download Android Beta</Button>
+                <Button secondary hero nuxt to="/about">iOS Coming Soon</Button>
+              </div>
+
+            </div>
+          </v-col>
+        </v-row>
+
+
+        <!-- How It Works section -->
+        <v-row class="d-block mb-10 pa-12" id="how-it-works">
+          
+          <h2 class="plum--text mb-12">How it Works</h2>
```

Comment:
Should `<h2>` be plum by default? I think so, actually- we can switch that in default.vue